README.md

```diff
@@ -22,16 +24,29 @@ If you have never used nightly Rust, the official guide has
 [a great section](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html#rustup-and-the-role-of-rust-nightly)
 about installing it.
 
-PyPy is also supported (via cpyext) for Python 3.5 only, targeted PyPy version is 7.0.0.
-Please refer to the guide for installation instruction against PyPy.
+### OS dependent packages
+Basically, you only need `Python`.
+However, on some OSs, you need to install some additional packages including Python header files.
 
-You can either write a native Python module in Rust, or use Python from a Rust binary.
+E.g. if you are on *Ubuntu 18.04*, please run
+
+```bash
+sudo apt install python3-dev
+```
 
-However, on some OSs, you need some additional packages. E.g. if you are on *Ubuntu 18.04*, please run
+### pyenv
+[pyenv](https://github.com/pyenv/pyenv) is a major Python version manager.
 
+To use PyO3 with pyenv, please set `--enable--shared` when builiding CPython.
```

Comment:
Is this necessary on all platforms or only on mac? I'm using pyenv on ubuntu and I do not remember setting that option.

Reply:
I don't have a macOS machine, but weirdly, Ubuntu 18.04 + Python 3.7.6(from pyenv) works but Arch Linux + Python 3.7.6 doesn't work.
There's certainly some OS-specific issue, but I still haven't detect it :cold_sweat: :

Reply:
I guess in this it's best to just document this as "you might `--enable--shared` on some linux distributions"